general/array.hpp

```diff
@@ -91,10 +92,15 @@ class Array
    template <typename CT>
    inline Array(const Array<CT> &src);
 
-   /// Deep copy from a braced init-list of convertible type
+   /// Construct an Array from a C-style array of static length
    template <typename CT, int N>
    explicit inline Array(const CT (&values)[N]);
 
+   /// Construct an Array from a braced initializer list of convertible type
+   template <typename CT, typename std::enable_if<
+                std::is_convertible<CT,T>::value,bool>::type = true>
```

Comment:
minor: mfem-4.8 should support C++17 (based on Tzanio's _state of mfem_ workshop slides) ; if we can already use it maybe ` std::enable_if_t<std::is_convertible_v<CT, T>>` might be simpler.

Reply:
@tzanio, thoughts?

Reply:
Since using C++11 is possible and not much more complicated, I'd say, let's stick with C++11 for this.